models/fedra.go

```diff
@@ -0,0 +1,87 @@
+package models
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/viper"
+	"github.com/vulsio/goval-dictionary/fetcher"
+)
+
+// ConvertFedoraToModel Convert OVAL to models
+func ConvertFedoraToModel(data *fetcher.FedoraUpdates) (defs []Definition) {
+	for _, update := range data.UpdateList {
+		if strings.Contains(update.Description, "** REJECT **") {
+			continue
+		}
+
+		cves := []Cve{}
+		for _, cveID := range update.CVEIDs {
+			cves = append(cves, Cve{
+				CveID: cveID,
+				Href:  fmt.Sprintf("https://cve.mitre.org/cgi-bin/cvename.cgi?name=%s", cveID),
+			})
+		}
+
+		packs := []Package{}
+		for _, pack := range update.Packages {
+			packs = append(packs, Package{
+				Name:            pack.Name,
+				Version:         fmt.Sprintf("%s:%s-%s", pack.Epoch, pack.Version, pack.Release),
+				Arch:            pack.Arch,
+				ModularityLabel: update.ModularityLabel,
+			})
+		}
+
+		refs := []Reference{}
+		bs := []Bugzilla{}
+		for _, ref := range update.References {
+			refs = append(refs, Reference{
+				Source: ref.Type,
+				RefID:  ref.ID,
+				RefURL: ref.Href,
+			})
+			if ref.Type == "bugzilla" {
+				bs = append(bs, Bugzilla{
+					BugzillaID: ref.ID,
+					URL:        ref.Href,
+					Title:      ref.Title,
+				})
+			}
+		}
+
+		issuedAt, _ := time.Parse("2006-01-02 15:04", update.Issued.Date)
+		updatedAt, _ := time.Parse("2006-01-02 15:04", update.Updated.Date)
```

Comment:
If we pick up the err here, we may notice when the datetime layout of updateinfo.xml changes?